packages/apps/transfer/src/components/Global/OptionsModal/styles.css.ts

```diff
@@ -0,0 +1,40 @@
+import { sprinkles } from '@kadena/react-ui/theme';
+
+import { style } from '@vanilla-extract/css';
+
+export const modalOptionsContent = style([
```

Comment:
I think we have to postfix all these exported consts with `Style`, so `modalOptionsContentStyle`